src/components/Comment/Comment.js

```diff
@@ -146,7 +152,13 @@ class Comment extends Component {
           {...(collapsed ? styles.bodyCollapsed : undefined)}
           style={{opacity: published ? 1 : 0.5}}
           ref={this.commentBodyRef}>
-          {!!content && renderMdast(content, schema, { MissingNode })}
+          {context && context.title && (
+            <div {...styles.context}>
+              <CommentContext {...context} />
+            </div>
+          )}
+          {!!content && !children && renderMdast(content, schema, { MissingNode })}
```

Comment:
`{children && children}` had the same end result as `{children}`.

A suggestion to simplify the next two line: 
```suggestion
          {children || (!!content && renderMdast(content, schema, { MissingNode }))}
```
_watchout GH suggestions can't replace two lines as far as I know._